### DISCUSSION TITLE
Trouble with disk mounting

### DISCUSSION THREAD

**Discussion by walksanatora** [microsoft/WSL#9935] [Q&A]
so I am trying to mount my linux partition
but when I get 
```
The disk is in use or locked by another process.
Error code: Wsl/Service/AttachDisk/0x8007006c
```
when I check `wmic partition list brief` I see
```
BootPartition  Index  Name                   NumberOfBlocks  PrimaryPartition  Size
FALSE          0      Disk #1, Partition #0  244240384       TRUE              125051076608
TRUE           0      Disk #0, Partition #0  1046528         TRUE              535822336
FALSE          1      Disk #0, Partition #1  2048            FALSE             1048576
FALSE          2      Disk #0, Partition #2  1952473088      FALSE             999666221056
```

but Disk 0 Partition 1 should not be the boot drive as that is where GRUB is but i booted directly using the device selector
should I try changing it in the BIOS and then rebooting? nope that did not fix it, it is still labeled as boot partition despite never touching grub during the boot process

**Comment by cachius** [comment]
Make sure to set the disk offline via [diskmgmt.msc](https://github.com/microsoft/WSL/issues/5997#issuecomment-1064557755) or [diskpart](https://answers.microsoft.com/en-us/windows/forum/all/online-offline-a-disk-in-disk-management-w10-home/c4c472a8-7fa1-41a5-a9ad-68434b64dc35#threadReplies):

```
Offline/online a disk with CMD:

1-Start > Type CMD > Right-click on it and Run as administrator
2-Type this command: diskpart and hit enter
3-Type this command: list disk and hit enter
4-Type this command: select disk 0 or 1 (select the disk that you want to be offline) and hit enter
5- Type this command: Offline or Online disk and hit enter.
```
Then from an elevated CMD/powershell run `wsl --mount --bare \\.\PHYSICALDRIVE0`
